tests/apps/configurationapp/app.go

```diff
@@ -0,0 +1,274 @@
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/dapr/components-contrib/configuration"
+	"github.com/dapr/components-contrib/tests/utils/configupdater"
+	cu_redis "github.com/dapr/components-contrib/tests/utils/configupdater/redis"
+	"github.com/dapr/dapr/tests/apps/utils"
+	"github.com/dapr/kit/logger"
+	"github.com/gorilla/mux"
+)
+
+const (
+	daprHost        = "localhost"
+	daprHTTPPort    = "3500"
+	configStoreName = "configstore"
+	redisHost       = "dapr-redis-master.dapr-tests.svc.cluster.local:6379"
+)
+
+var (
+	appPort         = 3000
+	lock            sync.Mutex
+	receivedUpdates map[string][]string
+	updater         configupdater.Updater
+)
+
+type testCommandRequest struct {
+	Message string `json:"message,omitempty"`
+}
+
+type appResponse struct {
+	Message   string `json:"message,omitempty"`
+	StartTime int    `json:"start_time,omitempty"`
+	EndTime   int    `json:"end_time,omitempty"`
+}
+
+type receivedMessagesResponse struct {
+	ReceivedUpdates []string `json:"received-messages"`
+}
+
+func init() {
+	p := os.Getenv("PORT")
+	if p != "" && p != "0" {
+		appPort, _ = strconv.Atoi(p)
+	}
+	receivedUpdates = make(map[string][]string)
+}
+
+// indexHandler is the handler for root path
+func indexHandler(w http.ResponseWriter, r *http.Request) {
+	log.Println("indexHandler is called")
+
+	w.WriteHeader(http.StatusOK)
+	json.NewEncoder(w).Encode(appResponse{Message: "OK"})
+}
+
+// sendResponse returns response with status code and message
+func sendResponse(w http.ResponseWriter, statusCode int, message string) {
+	w.WriteHeader(statusCode)
+	json.NewEncoder(w).Encode(appResponse{Message: message})
+}
+
+// getKeyValues is the handler for getting key-values from config store
+func getKeyValues(w http.ResponseWriter, r *http.Request) {
+	var keys []string
+	err := json.NewDecoder(r.Body).Decode(&keys)
+	if err != nil {
+		sendResponse(w, http.StatusBadRequest, err.Error())
+		return
+	}
+	url := "http://" + daprHost + ":" + daprHTTPPort + "/v1.0-alpha1/configuration/" + configStoreName + buildQueryParams(keys)
+	resp, err := http.Get(url)
+	if err != nil {
+		sendResponse(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+	defer resp.Body.Close()
+	respInBytes, _ := ioutil.ReadAll(resp.Body)
+	sendResponse(w, http.StatusOK, string(respInBytes))
+}
+
+func buildQueryParams(keys []string) string {
+	if len(keys) == 0 {
+		return ""
+	}
+	var ret string = "?key=" + keys[0]
+	for i := 1; i < len(keys); i++ {
+		ret = ret + "&key=" + keys[i]
+	}
+	return ret
+}
+
+func subscribeToConfigUpdates(keys []string) (string, error) {
+	url := "http://" + daprHost + ":" + daprHTTPPort + "/v1.0-alpha1/configuration/" + configStoreName + "/subscribe" + buildQueryParams(keys)
+	resp, err := http.Get(url)
+	if err != nil {
+		return "", fmt.Errorf("Error subscribing config updates: %s", err)
```

Comment:
In Go, errors always start lowercase (i'm surprised the linter hasn't complained!)


```suggestion
		return "", fmt.Errorf("error subscribing config updates: %w", err)
```